mlir/lib/Conversion/ConvertToSPIRV/ConvertToSPIRVPass.cpp

```diff
@@ -37,20 +37,53 @@ using namespace mlir;
 namespace {
 
 /// A pass to perform the SPIR-V conversion.
-struct ConvertToSPIRVPass final
-    : impl::ConvertToSPIRVPassBase<ConvertToSPIRVPass> {
+class ConvertToSPIRVPass
+    : public impl::ConvertToSPIRVPassBase<ConvertToSPIRVPass> {
```

Comment:
why change to class?

Reply:
I got some compile errors after introducing the new command-line option so I copied it from other passes, but I can change it back.